src/Components/Confirmation/Confirmation.js

```diff
@@ -19,16 +19,17 @@ import { getStepDirectory, getStepNumber, STARTING_QUESTION_NUMBER } from '../..
 import { useContext, useEffect } from 'react';
 import { Context } from '../Wrapper/Wrapper.tsx';
 import Grid from '@mui/material/Grid';
-import EditIcon from '@mui/icons-material/Edit';
-import HomeIcon from '@mui/icons-material/Home';
-import MedicalServicesIcon from '@mui/icons-material/MedicalServices';
-import ChecklistIcon from '@mui/icons-material/Checklist';
-import FoodBankIcon from '@mui/icons-material/FoodBank';
-import PeopleIcon from '@mui/icons-material/People';
-import PersonIcon from '@mui/icons-material/Person';
-import PaymentsIcon from '@mui/icons-material/Payments';
-import SavingsIcon from '@mui/icons-material/Savings';
-import ConnectWithoutContactIcon from '@mui/icons-material/ConnectWithoutContact';
+
+import Edit from '../../Assets/edit.svg';
```

Comment:
Could you import these as components? You then would not have to use them in an `img` tag. This will also let us change the color of the icons based on the style controller. More info [here](https://stackoverflow.com/a/52605225).
```suggestion
import { ReactComponent as Edit } from '../../Assets/edit.svg';
```

Reply:
Resolved console errors and updated svg imports.

![image](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/bbe745f5-8a2a-4a8f-b7af-0c58d375ae5f)